### PR TITLE
Reset price and time scale double click options

### LINKED ISSUE #1118: Reset price and time scale double click options
**Is your feature request related to a problem? Please describe.**

Now we have one option for resetings the price scales and the time scale

https://github.com/tradingview/lightweight-charts/blob/f28f8112c43f6cc1f0a953dc5dd5ec335af4b39b/src/model/chart-model.ts#L91-L96

**Describe the solution you'd like**

I suggest to replace this option with the two new options:

```TS
/** 
 * Enable resetting scaling the price scales by double-clicking the left mouse button. 
 * 
 * @defaultValue `true` 
 */ 
priceAxisDoubleClickReset: boolean; 


/** 
 * Enable resetting scaling the time scale by double-clicking the left mouse button. 
 * 
 * @defaultValue `true` 
 */ 
timeAxisDoubleClickReset: boolean; 
```

### PR DESCRIPTION
**Pull request by kirchet**
**Type of PR:** enhancement

**PR checklist:**

- [x] Addresses an existing issue: fixes #1118
- [ ] Includes tests
- [x] Documentation update

**Overview of change:**

https://github.com/tradingview/lightweight-charts/blob/65785ffb24f2a8b17dd7908a943ebd6cec405cce/src/model/chart-model.ts#L145-L162

https://github.com/tradingview/lightweight-charts/blob/65785ffb24f2a8b17dd7908a943ebd6cec405cce/src/model/chart-model.ts#L91-L95

### CODE REVIEW

File: src/api/options/chart-options-defaults.ts [hunk 1]
```diff
@@ -43,7 +43,10 @@ export const chartOptionsDefaults: ChartOptionsInternal = {
 			time: true,
 			price: true,
 		},
-		axisDoubleClickReset: true,
+		axisDoubleClickReset: {
```
Comment (eugene-korobko) [inline src/api/options/chart-options-defaults.ts:46]: Should we reflect back compatibility in this typing?

Comment (kirchet) [reply src/api/options/chart-options-defaults.ts:46]: Didn't understand. What do you mean?

Comment (eugene-korobko) [reply src/api/options/chart-options-defaults.ts:46]: Well, I've checked - resulting typing contains both boolean and new structure

**Review by eugene-korobko — APPROVED** [review]

**Review by eugene-korobko — APPROVED** [review]

**Comment by MarvinMiles** [comment]
Hello!

Can I reset priceScale programmatically? Can't find available API method for that.

**Pull request merged**